app/controllers/api/v1/admin/server_rooms_controller.rb

```diff
@@ -14,7 +14,17 @@ def index
 
           pagy, rooms = pagy(rooms)
 
-          rooms = RunningMeetingChecker.new(rooms:).call
+          active_rooms = BigBlueButtonApi.new.active_meetings
+          active_rooms_hash = {}
+
+          active_rooms.each do |active_room|
+            active_rooms_hash[active_room[:meetingID]] = active_room[:participantCount]
+          end
+
+          rooms.each do |room|
+            room.active = active_rooms_hash.key?(room.meeting_id)
+            room.participants = active_rooms_hash[room.meeting_id]
+          end
```

Comment:
I put the old ServerRooms logic so the specs doesn't break.
I think `active_rooms` is just running a bunch of `get_meeting_info`, so we might as well use our service in ServerRooms as well?